README.md

```diff
@@ -28,7 +28,14 @@ The following parameters can be used:
 
 ## 🌱 Prerequisites
 
-No prerequisites are required to run this executor. 
+To install the dependencies locally, run 
```

Comment:
Can you remove the` , `before `run`?

Reply:
yup https://github.com/jina-ai/executor-transformer-torch-encoder/pull/5/commits/fca885975dbcd90f084d56057bdd320249a5510f